projects/components/src/datagrid/datagrid.component.spec.ts

```diff
@@ -267,7 +263,7 @@ describe('DatagridComponent', () => {
                             pageSizeOptions: [10],
                         };
                         this.finder.detectChanges();
-                        expect(this.clrGridWidget.getPaginationDescription()).toEqual('1 - 51 of 150 items');
+                        expect(this.clrGridWidget.getPaginationDescription()).toEqual('1 - 53 of 150 items');
```

Comment:
Fixed the test for the one that uses the buttons (it seems like it was a little broken before), but the test for the others needs to change. This is because `clr-dg-footer` contains a few items that wrap on the line in testing style (they don't with proper style). In Angular 8/Clarity 2, footer.clientHeight was smart enough to calculate the correct height, but now it doesn't seem to know that the line has wrapped. Probably due to the testing issue you and I were talking about.